test/unit/amqpBinding-test3.js

```diff
@@ -0,0 +1,223 @@
+/*
```

Comment:
instead of -test3.js it would be better a more meaningful suffix, according to the kind of test cases included in this file.

Reply:
Maybe test/unit/amqpBinding-vhost-test.js ?

Reply:
Nice! I just followed what I found there. Name it as you like.